packages/editor/src/components/post-preview-button/index.js

```diff
@@ -194,6 +194,9 @@ export class PostPreviewButton extends Component {
 			<Button
 				isSecondary={ ! this.props.className }
 				className={ classNames }
+				height={ 50 }
```

Comment:
These changes are only for demonstrating how to overwrite the variant styles (`isSecondary` in that case) with styled system props